isUnique/isUnique.js

```diff
@@ -1,11 +1,19 @@
-/* 
+/*
 * Implement an algorithm to determine if a string has all unique characters.
 * Extra Credit - Answer this question - What if you cannot use additional data structures?
 */
 
 const isUnique = (str) => {
-
+  // for (initialize variable; condition to terminate; iteration step)
+  for (let i = 1; i < str.length; i++) {
+    // console.log(`index value: ${i} & string letter: ${str[i]}`);
```

Comment:
Great testing piece here. This solution is one of my favorite ways to solve this problem. You have a single loop and you're just passing once over the array! Nice job!